tests/indent/smali_spec.lua

```diff
@@ -0,0 +1,25 @@
+local Runner = require("tests.indent.common").Runner
+
+local run = Runner:new(it, "tests/indent/smali", {
+  tabstop = 4,
+  shiftwidth = 4,
+  expandtab = false,
+})
+
+describe("indent Smali:", function()
+  describe("whole file:", function()
+    run:whole_file(".", {
+      expected_failures = {},
+    })
+  end)
+
+  describe("new line:", function()
+    run:new_line("field.smali", { on_line = 7, text = 'value1 = "test"', indent = 1 })
+    run:new_line("field.smali", { on_line = 10, text = "value2 = Lsome/enum;", indent = 2 })
+    run:new_line("array_and_switch.smali", { on_line = 43, text = "1 2 3 4 5 6 200", indent = 2 })
+    run:new_line("array_and_switch.smali", { on_line = 48, text = "Label10:", indent = 2 })
+    run:new_line("method.smali", { on_line = 7, text = ".registers 10", indent = 1 })
+    run:new_line("parameter.smali", { on_line = 20, text = ".annotation runtime Lsome/annotation;", indent = 2 })
```

Comment:
Running tests, this line is breaking but im not sure if the intended indentation here is 2 or 3

Reply:
it should be 2

Reply:
Curious why, because the line above it is `.annotation ....` too, which means it should be indented one more to accomodate for that